src/inc/quic_platform_linux.h

```diff
@@ -633,6 +633,29 @@ typedef struct QUIC_THREAD_CONFIG {
     void* Context;
 } QUIC_THREAD_CONFIG;
 
+#ifdef QUIC_USE_CUSTOM_THREAD_CONTEXT
+
+//
+// Extension point that allows additional platform specific logic to be executed
+// for every thread created. The platform must define QUIC_USE_CUSTOM_THREAD_CONTEXT
+// and implement the QuicThreadCustomStart function. QuicThreadCustomStart MUST
+// call the Callback passed in. QuicThreadCustomStart MUST also free
+// CustomContext before returning.
```

Comment:
Should probably also say must be freed with `QUIC_FREE(CustomContext, QUIC_POOL_CUSTOM_THREAD);` that way you don't have to dig into the impl to figure that out.